Replace Lorem text in main activity with something meaningful

waqarsoomro9714 opened the pull request:
completed the task "Replace Lorem text in main activity with something meaningful" changed it to take care of your loved onces.

waqarsoomro9714 committed:
changes on main activity

completed the task "Replace Lorem text in main activity with something meaningful" changed it to take care of your loved onces.

waqarsoomro9714 commented:
please review my code thanks.

el-veee commented:
Thanks for using a string resource. 

Couple points of feedback. 
- Can you rename your PR to something more descriptive of the change you've made. You could use the title of the Trello card if you like
- You have a merge conflict to resolve

el-veee commented on app/src/main/res/values/strings.xml:
```diff
@@ -1,6 +1,8 @@
 <resources>
     <string name="app_name">TG_Patient_Profile</string>
 
+    <string name="slogan_on_main_activity">Take care of your loved onces</string>
```
I like most of this but I don't think "loved ones" makes sense for a patient management app to be used in a professional setting. 

Something like "Take Care of your Patients" is simple and to the point

waqarsoomro9714 committed:
migrated to model classes into kotlin

Changed the name of Medical_diagnostic model class to MedicalDignostic and converted to kotlin, also migrated the NextOfKin into Kotlin.

waqarsoomro9714 committed:
Replace Lorem text in main activity with something meaningful

"Replace Lorem text in main activity with something meaningful" changed it to We take care of your loved onces.

waqarsoomro9714 commented:
please check the code thank you.

el-veee commented:
You're starting to tackle a separate task in this PR. They should be separate PRs from separate branches.

waqarsoomro9714 committed:
Migrate Medical_diagnostic Model to Kotlin

Migrate Medical_diagnostic Model to Kotlin.

el-veee commented on app/src/main/res/values/strings.xml:
```diff
@@ -1,6 +1,8 @@
 <resources>
     <string name="app_name">TG_Patient_Profile</string>
 
+    <string name="slogan_on_main_activity">We take care of your loved onces</string>
```
Again, this app is meant to be used by professionals. The "loved ones" here would be patients. This app isn't a bridge between caregivers and patient's families so I don't think this text suits Guardian

el-veee commented:
You should also only tackle 1 model migration per PR. There's separate Trello cards for each of these. This is to reduce the risk of regressions

el-veee commented on app/src/main/java/com/example/tg_patient_profile/model/MedicalDiagnostic.kt:
Should not be in this PR

el-veee commented on app/src/main/java/com/example/tg_patient_profile/model/Medical_diagnostic.java:
Should not be in this PR

el-veee commented on app/src/main/java/com/example/tg_patient_profile/model/NextOfKin.java:
Should not be in this PR

el-veee commented on app/src/main/java/com/example/tg_patient_profile/model/NextOfKin.kt:
Should not be in this PR

el-veee commented on app/src/main/java/com/example/tg_patient_profile/view/general/PatientProfileAddActivity.java:
Should not be in this PR

el-veee commented on app/src/main/java/com/example/tg_patient_profile/view/patient/patientdata/medicaldiagnostics/CurrentMedicalDiagnosticsFragment.java:
Should not be in this PR

el-veee commented on app/src/main/java/com/example/tg_patient_profile/view/patient/patientdata/medicaldiagnostics/PastMedicalDiagnosticsFragment.java:
Should not be in this PR

el-veee reviewed (changes requested):
This PR should only contain changes related to updating the Main Activity text. You should remove the model changes, and create separate PRs for them. You can uses branching for this.

waqarsoomro9714 committed:
Take Care of your Patients

Updated the text on home page.
